### DISCUSSION TITLE
Supported Kubeflow Versions

### DISCUSSION THREAD

**Discussion by ozen** [elyra-ai/elyra#3091] [Q&A]
Hello all,

I have been digging into Elyra for the last couple of days and I think it would be very useful for my team. But I'm surprised to see no references/talks/issues etc. about Kubeflow versions that we can use with Elyra, except the following text from the docs:

```
Note: Elyra is only tested with Kubeflow v1.2.x and v1.3.x and Apache Airflow v1.10.x.
```

There has been 3 major KF releases since 1.3:
1.4 on October 2021
1.5 on March 2022
1.6 on September 2022
and 1.7 is planned for March 2023

KF has been evolving rapidly, so the users are keen to upgrade.

I wonder if the Elyra team can do more to test the newer versions, @akchinSTC

I also wonder if any Elyra users here successfully run Elyra with the newer versions of KF.

**Comment by akchinSTC** [comment]
Hi @ozen, 

Thanks for opening up a discussion. As a team, we have spent most of the latter half of last year implementing the framework to better support kubeflow components and so testing with newer versions of the kubeflow 1.x have been slower than we like. With the finite amount of resources our team has, our plan for the new year was to get a jump on Elyra 4.x and with that, support for Jupyter lab 4.x and KFP 2.x SDK. 

With that said, we *may*(a big may, given our current commitments) be able to squeeze in some token testing against the newer versions (up to KF v1.5.1, 1.6.0 uses the new v2 backend). Like always we welcome any testing/contributions/observations you may have done in the meantime.

**Reply by ptitzler** [reply]
It appears that the docs are outdated. We are currently testing using KF 1.5.1.

Edit: https://github.com/elyra-ai/elyra/pull/3094 updates the doc
